composer.json

```diff
@@ -32,7 +32,7 @@
         {"name": "Jonathan Wage", "email": "jonwage@gmail.com"}
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.3 || ^8",
```

Comment:
```suggestion
        "php": "^7.3 || (^8 && <8.1)",
```

Reply:
@mvorisek please make sure you provide an explanation if you're suggesting a change.

Reply:
Add explicit support for php 8.0.x only, we should not declare like 8.1 support if not tested against this version (with which this core package might be incompatible at the current state)

Reply:
We haven't tested against php 7.5 either, yet we declare compatibility. I think we're safe here.